test/test_definitions.py

```diff
@@ -2,33 +2,37 @@
 Resource definitions and helper functions for tests.
 '''
 import shutil
-import tempfile
 import unittest
+import os
 # Try python3 dependency, fall back if not available
 try:
     from urllib2 import urlopen
 except ImportError:
     from urllib.request import urlopen
+from cve_bin_tool.NVDAutoUpdate import DISK_LOCATION_DEFAULT
 
 CURL_7_20_0_RPM = 'curl-7.20.0-4.fc13.x86_64.rpm'
+VMWARE = 'vmware-1.1.cab'
+TMUX = 'tmux_1.8-5_amd64.deb'
 CURL_7_20_0_URL = 'https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/13/Everything/x86_64/os/Packages/' + CURL_7_20_0_RPM
-VMWARE_CAB = 'https://master.dl.sourceforge.net/project/winpe/VmWare%20Drivers/VmWare%20Drivers%20v1.1/vmware-1.1.cab'
-TMUX_DEB = 'https://mirrors.cat.pdx.edu/ubuntu/pool/main/t/tmux/tmux_1.8-5_amd64.deb'
+VMWARE_CAB = 'https://master.dl.sourceforge.net/project/winpe/VmWare%20Drivers/VmWare%20Drivers%20v1.1/' + VMWARE
+TMUX_DEB = 'https://mirrors.cat.pdx.edu/ubuntu/pool/main/t/tmux/' + TMUX
 
 class TempDirTest(unittest.TestCase):
     """ For tests that need a temp directory """
 
     @classmethod
     def setUpClass(cls):
-        cls.tempdir = tempfile.mkdtemp()
-
-    @classmethod
-    def tearDownClass(cls):
-        shutil.rmtree(cls.tempdir)
+        """ Set location on disk where test data cache will reside."""
+        cls.tempdir = os.path.join(DISK_LOCATION_DEFAULT , 'test')
+        if not os.path.exists(cls.tempdir):
+            os.makedirs(cls.tempdir)
 
 def download_file(url, target):
     """ helper method to download a file """
-    download = urlopen(url)
-    with open(target, 'wb') as target_file:
+    if not os.path.exists(target): 
+        download = urlopen(url)
+        target_file = open(target, 'wb')
```

Comment:
please keep the `with` here, it will close the file for us if things go wrong (or just always when things go right). Then I think this is ready to go! Thanks!